### ISSUE TITLE
Extraneous props passed to select element, resulting in React warning

### ISSUE THREAD

**Issue by justin-perinovic** [skratchdot/react-bootstrap-multiselect#49]
This appears to be the offending code, in lib/index.js:

> render: function () {
		//this.setOptionsFromProps();
		return React.createElement('select',
			objectAssign({}, this.props, {ref: 'multiselect'}));
	}

**Comment by wvengen** [comment]
This is an example warning I'm seeing:
```
Warning: Unknown props `nonSelectedText`, `buttonClass` on <select> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in select (created by MultiSelect)
    in MultiSelect (created by FooView)
```

**Comment by skratchdot** [comment]
I just published a new version 2.3.0 that *should* fix this.  Please re-open if you still see the error.  In my local testing, 'data' did not throw the error, but I'm not sure if that's true for all versions of react 15

**Comment by wvengen** [comment]
I can confirm that this is solved (using React 15.3.2), thanks!
